web/src/beta/features/Editor/index.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta, StoryObj } from "@storybook/react";
```

Comment:
Can you write stories as shown [here](https://storybook.js.org/docs/react/writing-stories/introduction).
You can check [this component](https://github.com/reearth/reearth/blob/main/web/src/classic/components/atoms/Modal/index.stories.tsx) as an example (just don't add a `title` for beta components`